src/core/Cline.ts

```diff
@@ -1994,8 +1985,12 @@ export class Cline {
 										break
 									}
 								}
+								// Get context window and used context from API model
+								const contextWindow = this.api.getModel().info.contextWindow || 64_000 // minimum context (Deepseek)
```

Comment:
We should have this 64_000 as a constant somewhere.   We have this in quite a few places and might miss it during an update in a place here or there.

Reply:
Discussed this with @ocasta181 and we decided that this will be addressed in a future change.

Reply:
discussed offline